### DISCUSSION TITLE
Support DiceDB to replace Redis

### DISCUSSION THREAD

**Discussion by henriquekieckbusch** [wardenenv/warden#810] [Ideas]
It would be great to give support to https://github.com/dicedb/dice as an additional "redis" service.

You can replace Redis with it (they share the same connectors etc.).

All the benchmarks show it running at least 5 times faster than Redis.

**Comment by bap14** [comment]
@henriquekieckbusch  An interesting potential replacement; however, it's noted that it currently only supports a subset of the Redis commands. You are able to replace the redis service for whatever environment you want using the `<project_dir>/.warden/warden-env.yml` file. There doesn't need to be a Warden-specific build in order to use it with Warden.

That being said, I'd be for adding it as an alternative for Redis once it reaches 100% compatability.

**Comment by henriquekieckbusch** [comment]
I tested it, I had to do multiple changes to the Redis.php, so it is not yet a good option
